apps/web/src/components/PoolStatusTable/PoolStatusTable.tsx

```diff
@@ -26,200 +29,138 @@ const isSuspended = (suspendedAt: PoolCandidate["suspendedAt"]): boolean => {
   return isPast(suspendedAtDate);
 };
 
+const columnHelper = createColumnHelper<PoolCandidate>();
+
 const PoolStatusTable = ({ user, pools }: UserInformationProps) => {
   const intl = useIntl();
   const paths = useRoutes();
 
-  const poolCandidatesArray = user.poolCandidates?.filter(notEmpty);
+  const columns = [
+    columnHelper.accessor((row) => getFullPoolTitleLabel(intl, row.pool), {
+      id: "pool",
+      meta: {
+        isRowTitle: true,
+      },
+      enableHiding: false,
+      cell: ({ row: { original: candidate }, getValue }) =>
+        cells.view(paths.poolView(candidate.pool.id), getValue()),
+      header: intl.formatMessage({
+        defaultMessage: "Pool",
+        id: "icYqDt",
+        description:
+          "Title of the 'Pool' column for the table on view-user page",
+      }),
+    }),
+    columnHelper.accessor(
+      (row) => intl.formatMessage(getPoolCandidateStatus(row.status as string)),
+      {
+        id: "status",
+        enableHiding: false,
+        cell: ({ row: { original: candidate } }) =>
+          statusCell(candidate, user, pools),
+        header: intl.formatMessage({
+          defaultMessage: "Status",
+          id: "sUx3ZS",
+          description:
+            "Title of the 'Status' column for the table on view-user page",
+        }),
+        sortingFn: sortStatus,
+      },
```

Comment:
It appears it no longer needs that custom sort order 
https://github.com/GCTC-NTGC/gc-digital-talent/issues/7431#issuecomment-1755582480

Reply:
@Jerryescandon We had sorting based on the following order:

```tsx
const sortOrder = [
  PoolCandidateStatus.PlacedIndeterminate,
  PoolCandidateStatus.PlacedTerm,
  PoolCandidateStatus.PlacedCasual,
  PoolCandidateStatus.QualifiedAvailable,
  PoolCandidateStatus.QualifiedUnavailable,
  PoolCandidateStatus.QualifiedWithdrew,
  PoolCandidateStatus.UnderAssessment,
  PoolCandidateStatus.ScreenedIn,
  PoolCandidateStatus.ScreenedOutAssessment,
  PoolCandidateStatus.ScreenedOutNotInterested,
  PoolCandidateStatus.ScreenedOutNotResponsive,
  PoolCandidateStatus.ScreenedOutApplication,
  PoolCandidateStatus.ApplicationReview,
  PoolCandidateStatus.NewApplication,
  PoolCandidateStatus.Removed,
  PoolCandidateStatus.Draft,
  PoolCandidateStatus.DraftExpired,
  PoolCandidateStatus.Expired,
];
```

Should we persist this or change it to alphabetical as suggested in the comment?

Reply:
I have opinions, but not strong feelings about this. Here's my train of thought.

I think this order provides some value, but not critical value, while making it alphabetical improves A11y significantly.

I am also hoping we are slowly reducing the recruitment team's need to visit and work using this table, and for it to become a reference point instead of a core work component. At which point the forced order is almost meaningless.

@gobyrne thoughts?

Reply:
Decision from Gray: continue to use custom sort for now